services/web/server/tests/integration/computation/test_computation.py

```diff
@@ -31,11 +25,12 @@
 from simcore_service_webserver.security import setup_security
 from simcore_service_webserver.security_roles import UserRole
 from simcore_service_webserver.session import setup_session
-from simcore_service_webserver.users import setup_users
 from utils_assert import assert_status
 from utils_login import LoggedUser
 from utils_projects import NewProject
 
+current_dir = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent

```

Comment:
fixture or not fixture... I tended to make a fixture of everything. in the end that is the same I don't know what is clearer.
anyway the test looks much cleaner.

Reply:
I found myself in an interesting situation with the famous ``here`` fixture. Since fixtures are overriden, my ``here`` path was actually taken as input in another fixture that was not really "here" :-). Therefore I decided to come back to the old good ``current_dir`` in this case